src/Controllers/rovers.controllers.ts

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+import {JSONRovers, JSONCamera} from '../Models/interfaces';
+import {API_BASE, API_KEY} from '../Routes/server';
+
+export function rovers(req: any, res: any) {
```

Comment:
the function name doesn't describe what the function does - `getRovers` will be better, what do you think?